Signal/src/util/OWSBackupExportJob.m

```diff
@@ -133,8 +135,8 @@ - (void)start
                                progress:nil];
 
     __weak OWSBackupExportJob *weakSelf = self;
-    [self configureExport:^(BOOL success) {
-        if (!success) {
+    [self configureExport:^(BOOL configureExportSuccess) {
```

Comment:
Futures would have made this code more readable, but I always find working with PromiseKit to be so frustrating.

Reply:
Thanks, let's try that sometime.